frontend/cms-angular/src/app/components/admin/admin.component.html

```diff
@@ -0,0 +1,31 @@
+
+<body>
+  <div id="wrapper">
+     <!-- Sidebar -->
+     <div id="sidebar-wrapper">
+        <ul class="sidebar-nav nav-pills nav-stacked" id="menu">
+           <li class="active">
+              <a href="admin/login"> <span class="fa-stack fa-lg pull-left"></span> Dashboard</a>
+           </li>
+           <li>
+              <a href="/admin/register"><span class="fa-stack fa-lg pull-left"></span>Pages</a>
+           </li>
+           <li>
+              <a href="#"><span class="fa-stack fa-lg pull-left"></span>Add Article</a>
```

Comment:
te trzy a hrefy powinny przekierowywać na wskazane strony, w naszym przypadku będzie to:
dla Articles /admin/articles
dla Templatek /admin/template
Dodawanie artykułu zrobimy na stronie z artykułami więc to można byłoby odpuścić. 
Dashboard hmmm, może go przemianować na Your Page i przenosić ze strony admina na home page? Co myślisz?